shared/constants/people.tsx

```diff
@@ -216,7 +216,50 @@ export const reduceRPCItemToPeopleItem = (
           ),
           notificationTime,
           numAdditional: multiFollow.numOthers,
-          type: 'notification',
+          type: 'follow',
+        })
+      )
+    } else if (notification && notification.t === RPCTypes.HomeScreenPeopleNotificationType.contact) {
+      // Single contact notification
+      const follow = notification.contact
+      if (!follow) {
+        return list
+      }
+      return list.push(
+        makeFollowedNotificationItem({
+          badged,
+          newFollows: [
+            makeFollowedNotification({contactDescription: follow.description, username: follow.username}),
+          ],
+          notificationTime: new Date(follow.resolveTime),
+          type: 'contact',
+        })
+      )
+    } else if (notification && notification.t === RPCTypes.HomeScreenPeopleNotificationType.contactMulti) {
+      // Multiple follows notification
+      const multiContact = notification.contactMulti
+      if (!multiContact) {
+        return list
+      }
+      const contacts = multiContact.contacts
+      if (!contacts) {
+        return list
+      }
+      const notificationTimes = contacts.map(contact => contact.resolveTime)
+      const maxNotificationTime = Math.max(...notificationTimes)
```

Comment:
can the go side just plumb this through

Reply:
this is how the multiple-follows logic works, I just copied that. The go side doesn't really touch this, just passes along what the iced side makes.

Reply:
That is to say, I could add it, but if I was gonna add it I'd also want to add it for the followers, and it involve more plumbing than I feel like doing right now.